src/CardShuffler/CardShuffler.test.js

```diff
@@ -0,0 +1,26 @@
+import { CardShuffler } from './CardShuffler.js';
+import { ImageLoader } from '../ImageLoader/ImageLoader.js';
+import '@testing-library/jest-dom';
+
+describe("CardShuffler", () => {
+  let cardList;
+  beforeEach(() => {
+    cardList = ImageLoader.map((image, i) => {return {img: image, index: i}});
+  })
+
+  it('should shuffle the cards', () => {
+    
+    CardShuffler.wash(cardList);
+    console.log(cardList[0].img);
+
+    expect(cardList[0].img).not.toStrictEqual({"img": "AS.png", "index": 0});
```

Comment:
It should be noted that this has a 1 in 52 chances of failing. Since this is smaller than a 2% chance of error, I left it in.